js/phoenix/src/pages/signup.jsx

```diff
@@ -0,0 +1,41 @@
+import { SignupForm } from "../containers";
+import { BareIcon } from "../../src/components"
+import { BsLinkedin, BsGoogle, BsGithub } from "react-icons/bs";
+
+const Signup = () => {
+    return (
+        <div className="flex flex-row">
+            <div className="basis-1 md:basis-2/3  flex flex-col justify-center items-center">
+                <h1 className="font-bold text-3xl">Add Details to Signup</h1>
+                <SignupForm />
+                <div className="flex items-center">
+                    <div className="border-t-2 border-gray-400  w-40"></div>
```

Comment:
@AbhiGattineni separate this into a component

Reply:
Separated and updated in the signup, and login pages.